### PR TITLE
49 print css

### PR DESCRIPTION
**Pull request by matthopson**
Adds Docket Record print view.

Couple of notes:

- There is some odd behavior from the mobile styles when the print preview is captured, catching some contents in transition, making them invisible, so things like removing <Mobile> were necessary to try and prevent this.
- The print view is a composed view, but some components still need some things hidden, which spreads that logic out. In order to clean this up better, we may need to rebuild the view from scratch without including those components. I'm concerned that modifying these components will introduce side-effects in the printed view if they aren't explicitly tested.
- Redirecting to the print view and then back to the referring view is not ideal. I tried modifying the current view, but I couldn't get the control over the styles to eliminate the odd behavior mentioned above. I also tried using an overlay to put the printed view above the current view. This may still be an option, but I was having some with this in the print preview, as elements below would spill out under. Perhaps hiding that content, along with the overlay is a solution to consider.

### CODE REVIEW

**Comment by ericsorenson** [comment]
pa11y caught a couple duplicate ID issues

```Errors in http://localhost:1234/mock-login?token=petitionsclerk&path=/case-detail/102-19&info=tab-case-info:

 • Duplicate id attribute value "primary-label" found on the web page.

   (#primary-label)

   <p class="label" id="primary-label">Primary Contact</p>

 • Duplicate id attribute value "secondary-label" found on the web page.

   (#secondary-label)

   <p class="label" id="secondary-label">Secondary Contact</p>```

File: web-client/src/views/CaseDetailHeader.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const CaseDetailHeader = connect(
   }) => {
     return (
       <div className="big-blue-header">
-        <div className="grid-container foo-bar">
```
Comment (ericsorenson) [inline web-client/src/views/CaseDetailHeader.jsx:26]: 🤭

File: web-client/src/views/CaseDetail/PartyInformation.jsx [hunk 2]
```diff
@@ -154,18 +153,18 @@ export const PartyInformation = connect(
     };
     return (
       <div className="subsection party-information">
-        <NonMobile>
+        <div className="only-large-screens">
```
Comment (ericsorenson) [inline web-client/src/views/CaseDetail/PartyInformation.jsx]: 👍

File: web-client/src/styles/custom.scss [hunk 3]
```diff
@@ -15,6 +15,18 @@ hr {
   margin: 20px 0;
 }
 
+.text-align-left {
```
Comment (ericsorenson) [inline web-client/src/styles/custom.scss:18]: This looks like something that should have been in USWDS, but you're right, it's not in there.

File: web-client/src/presenter/actions/printViewAction.test.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      },
+    });
+
+    setTimeout(() => {
+      expect(result.state.currentPage).toEqual('testView');
+    });
+  });
+  it('should call window.print()', async () => {
+    global.window = {
+      print: mockPrint,
```
Comment (ericsorenson) [inline web-client/src/presenter/actions/printViewAction.test.js:41]: 👏

File: web-client/src/presenter/computeds/caseDetailHelper.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+import { Case } from '../../../../shared/src/business/entities/cases/Case';
```
Comment (ericsorenson) [inline web-client/src/presenter/computeds/caseDetailHelper.js]: We should be getting entites from the applicationContext, which should be the second argument to every computed

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/computeds/caseDetailHelper.js (hunk 5 above)
Comment (matthopson) [reply web-client/src/presenter/computeds/caseDetailHelper.js]: Ah, you're right, I meant to go back and fix this.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**